src/main/kotlin/NullAbilityUpdated/Keren/Q4.kt

```diff
@@ -0,0 +1,25 @@
+package NullAbilityUpdated.Keren
+/**
+ * Modify the main function to iterate over the employees list and print each employee's city.
+ * Use safe calls (?.) and the Elvis operator (?:) to ensure that if an employee or their address is null,
+ * or if the city is null, "City Unknown" is printed instead.
+ * Try to write the solution in the most concise way possible using Kotlin's chaining capabilities.
+ *
+ **/
+data class Address(val city: String?, val street: String?)
+data class Employee(val name: String?, val address: Address?)
+
+fun main() {
+    val employees = listOf(
+        Employee("John", Address("New York", "Fifth Ave")),
+        Employee("Jane", null),
+        Employee(null, Address(null, "Unknown Street")),
+        Employee("Alice", Address("Los Angeles", null))
+    )
+
+    // Task: Print each employee's city safely. If the city is not available, print "City Unknown".
+    employees.printsAllEmployeesCites()
+}
+
+fun List<Employee>.printsAllEmployeesCites() = this.forEach( { println(it.address?.city ?: "City Unknown") })
```

Comment:
This shouldn't be an extension function on  List<Employee>, can be a regular function

Reply:
Got it
Yes, Keren, I know it shouldn’t be an extension—I was just practicing. 
Is there anything else you think could be improved aside from the unnecessary extension? 
Thanks again for the review!